### DISCUSSION TITLE
Alist挂载115网盘映射到本地后使用搜索功能会被115阻断

### DISCUSSION THREAD

**Discussion by hujingyi1993** [AlistGo/alist#5466] [General]
如题，因为资源太多，有时候要搜索专门的资源名称，一旦使用搜索（重建索引）后
就会产生如下报错

failed get objs: failed to list objs: <!DOCTYPE html> <html lang="zh-cn"> <head> <meta charset="utf-8"> <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"> <meta name="data-spm" content="a3c0e" /> <title>405</title> <style> html, body, div, a, h2, p { margin: 0; padding: 0; } a { text-decoration: none; color: #3b6ea3; } .container { width: 1000px; margin: auto; color: #696969; } .header { padding: 50px 0; } .header .message { height: 36px; padding-left: 120px; background: url(https://errors.aliyun.com/images/TB1TpamHpXXXXaJXXXXeB7nYVXX-104-162.png) no-repeat 0 -128px; line-height: 36px; } .main { padding: 50px 0; background: #f4f5f7; } .main img { position: relative; left: 120px; } .footer { margin-top: 30px; text-align: right; } .footer a { padding: 8px 30px; border-radius: 10px; border: 1px solid #4babec; } .footer a:hover { opacity: .8; } .alert-shadow { display: none; position: absolute; top: 0; left: 0; width: 100%; height: 100%; background: #999; opacity: .5; } .alert { display: none; position: absolute; top: 200px; left: 50%; width: 600px; margin-left: -300px; padding-bottom: 25px; border: 1px solid #ddd; box-shadow: 0 2px 2px 1px rgba(0, 0, 0, .1); background: #fff; font-size: 14px; color: #696969; } .alert h2 { margin: 0 2px; padding: 10px 15px 5px 15px; font-size: 14px; font-weight: normal; border-bottom: 1px solid #ddd; } .alert a { display: block; position: absolute; right: 10px; top: 8px; width: 30px; height: 20px; text-align: center; } .alert p { padding: 20px 15px; } </style> </head> <body data-spm="7663354"> <div data-spm="1998410538"> <div class="header"> <div class="container"> <div class="message"> 很抱歉，由于您访问的URL有可能对网站造成安全威胁，您的访问被阻断。 <div>您的请求ID是: <strong> 784e2c9616990311015157507e08fd</strong></div> </div> </div> </div> <div class="main"> <div class="container"> <img src="https://errors.aliyun.com/images/TB15QGaHpXXXXXOaXXXXia39XXX-660-117.png" /> </div> </div> <div class="footer"> <div class="container"> <a target="_blank" id="report" href="javascript:;" data-spm-click="gostr=/waf.123.123;locaid=d001;">误报反馈</a> </div> </div> </div> <div id="alertShadow" class="alert-shadow"></div> <div id="alertContainer" class="alert"> <h2>提示：<a href="javascript:;" title="关闭" id="closeAlert">X</a></h2> <p>感谢您的反馈，应用防火墙会尽快进行分析和确认。</p> </div> <script> function show() { var g = function(ele) { return document.getElementById(ele); }; var reportHandle = g('report'); var alertShadow = g('alertShadow'); var alertContainer = g('alertContainer'); var closeAlert = g('closeAlert'); var own = {}; own.report = function() { // SPM own.alert(); }; own.alert = function() { alertShadow.style.display = 'block'; alertContainer.style.display = 'block'; }; own.close = function() { alertShadow.style.display = 'none'; alertContainer.style.display = 'none'; }; }; </script> <script type="text/javascript" charset="utf-8" src="https://errors.aliyun.com/error.js? 784e2c9616990311015157507e08fd"></script> </body> </html>: unexpected error

**Comment by hankqqu** [comment]
> 如题，因为资源太多，有时候要搜索专门的资源名称，一旦使用搜索（重建索引）后
> 
> 就会产生如下报错
> 
> 
> 
> failed get objs: failed to list objs: <!DOCTYPE html> <html lang="zh-cn"> <head> <meta charset="utf-8"> <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"> <meta name="data-spm" content="a3c0e" /> <title>405</title> <style> html, body, div, a, h2, p { margin: 0; padding: 0; } a { text-decoration: none; color: #3b6ea3; } .container { width: 1000px; margin: auto; color: #696969; } .header { padding: 50px 0; } .header .message { height: 36px; padding-left: 120px; background: url(https://errors.aliyun.com/images/TB1TpamHpXXXXaJXXXXeB7nYVXX-104-162.png) no-repeat 0 -128px; line-height: 36px; } .main { padding: 50px 0; background: #f4f5f7; } .main img { position: relative; left: 120px; } .footer { margin-top: 30px; text-align: right; } .footer a { padding: 8px 30px; border-radius: 10px; border: 1px solid #4babec; } .footer a:hover { opacity: .8; } .alert-shadow { display: none; position: absolute; top: 0; left: 0; width: 100%; height: 100%; background: #999; opacity: .5; } .alert { display: none; position: absolute; top: 200px; left: 50%; width: 600px; margin-left: -300px; padding-bottom: 25px; border: 1px solid #ddd; box-shadow: 0 2px 2px 1px rgba(0, 0, 0, .1); background: #fff; font-size: 14px; color: #696969; } .alert h2 { margin: 0 2px; padding: 10px 15px 5px 15px; font-size: 14px; font-weight: normal; border-bottom: 1px solid #ddd; } .alert a { display: block; position: absolute; right: 10px; top: 8px; width: 30px; height: 20px; text-align: center; } .alert p { padding: 20px 15px; } </style> </head> <body data-spm="7663354"> <div data-spm="1998410538"> <div class="header"> <div class="container"> <div class="message"> 很抱歉，由于您访问的URL有可能对网站造成安全威胁，您的访问被阻断。 <div>您的请求ID是: <strong> 784e2c9616990311015157507e08fd</strong></div> </div> </div> </div> <div class="main"> <div class="container"> <img src="https://errors.aliyun.com/images/TB15QGaHpXXXXXOaXXXXia39XXX-660-117.png" /> </div> </div> <div class="footer"> <div class="container"> <a target="_blank" id="report" href="javascript:;" data-spm-click="gostr=/waf.123.123;locaid=d001;">误报反馈</a> </div> </div> </div> <div id="alertShadow" class="alert-shadow"></div> <div id="alertContainer" class="alert"> <h2>提示：<a href="javascript:;" title="关闭" id="closeAlert">X</a></h2> <p>感谢您的反馈，应用防火墙会尽快进行分析和确认。</p> </div> <script> function show() { var g = function(ele) { return document.getElementById(ele); }; var reportHandle = g('report'); var alertShadow = g('alertShadow'); var alertContainer = g('alertContainer'); var closeAlert = g('closeAlert'); var own = {}; own.report = function() { // SPM own.alert(); }; own.alert = function() { alertShadow.style.display = 'block'; alertContainer.style.display = 'block'; }; own.close = function() { alertShadow.style.display = 'none'; alertContainer.style.display = 'none'; }; }; </script> <script type="text/javascript" charset="utf-8" src="https://errors.aliyun.com/error.js? 784e2c9616990311015157507e08fd"></script> </body> </html>: unexpected error

**Reply by 2982136527** [reply]
如果你的 alist 内有挂载阿里云盘，可以将其卸载后重新启用 115 挂载，这只是我自己的解决办法，并不一定通用

**Comment by szhyf** [comment]
同样问题，貌似是115用了阿里云？

**Reply by sudikezhene** [reply]
我也是，用了VIDHUP这个APP

**Comment by kaneki-029** [comment]
我也是，大概 11 个 T 的影视资源，用 rclone 挂载在本地服务器，用 Plex 作为媒体服务器，每天晚上都会扫描一遍资料库。昨天被应用防火墙拦截了，不知道是不是因为访问太频繁导致的。

目前把 115 驱动 API 访问速率降到默认了（之前是 5/s） ，也把阿里云盘下掉了，rclone 本地缓存时间加长了点，再观察观察。
